web/src/main/scala/net/wiringbits/components/widgets/Footer.scala

```
@@ -89,17 +89,17 @@ import slinky.core.facade.Fragment
     val contact = Container(
       flex = Some(1),
       child = Fragment(
-        title("Contact"),
+        title(texts.contact),
         Container(
           child = Fragment(
-            subtitle("Email"),
+            subtitle(texts.contact),
             link("hello@wiringbits.net", "mailto:hello@wiringbits.net")
```

This is something I'd propagate in the context instead of hardcoding it.